noxfile.py

```diff
@@ -140,7 +140,7 @@ def integration_tests_boto3(
     session,
     boto3_version,
 ):
-    with TestedVersions.save_tests_result("boto3", "boto3", boto3_version):
+    with TestedVersions.save_tests_result("botocore", "boto3", boto3_version):
         install_package("boto3", boto3_version, session)
```

Comment:
```suggestion
        install_package("botocore", botocore_version, session)
```

Reply:
We are actually testing boto3 here, even if the instrumentation sits at botocore level. We should express our dependencies in terms of what the users are likely to import.